### DISCUSSION TITLE
Error : The type or namespace name localization' does not exist in the namespace 'Volo.Abp'

### DISCUSSION THREAD

**Discussion by ShifanilaSenthil** [abpframework/abp#18920] [Q&A]
Great product abp.io but we have some questions...

We have purchased abp commercial Business edition
1. Downloaded all the modules with the command abp add-module Volo.xxx --with-source-code
2. Replace the npm package reference with the reference of downloaded projects.
I tried to run the dbmigrator , but on Build it shows error like "The type or namespace name localization' does not exist in the namespace 'Volo.Abp'  and metadata file could not be found

**Comment by maliming** [comment]
hi

Please create a support ticket on https://support.abp.io/

Thanks.
